src/odemis/acq/stitching/_registrar.py

```diff
@@ -107,131 +114,138 @@ def addTile(self,tile,dependent_tiles = None):
         position, any following tile must either be inserted to the right of an existing tile or to
         the bottom. The grid has to be filled up from top to bottom, moving from the bottom up is not 
         supported.
-        """  
-        
+        """
+
         self.px_size = tile.metadata[model.MD_PIXEL_SIZE]
-        
+
         if dependent_tiles != None:
-            self.num_dep_tiles = len(dependent_tiles) 
-        
-        # Find position of the tile in the grid. Indices of grid position are given as posX, posY.
+            self.num_dep_tiles = len(dependent_tiles)
+
+        # Find position of the tile in the grid. Indices of grid position are
+        # given as posX, posY.
         if self.tiles[0][0] == None:
             self.size = tile.shape
             self.posX = 0
             self.posY = 0
             if dependent_tiles != None:
-            # Shift between tile and its dependent tiles
+                # Shift between tile and its dependent tiles
                 for i in range(len(dependent_tiles)):
-                    self.shift_tile_dep_tiles.append([])                   
+                    self.shift_tile_dep_tiles.append([])
 
-            self.pos_prev_x = 0 # used to determine if tile should be compared to right or left tile
+            self.pos_prev_x = 0  # used to determine if tile should be compared to right or left tile
         else:
             if tile.shape != self.tiles[0][0].shape:
                 raise ValueError(
```

Comment:
In such long error messages, I often just put the string on one line and the parameters on a second line, like this:
```
raise ValueError("Tile shape differs from previous tiles %s != %s" % 
                 (tile.shape, self.tiles[0][0].shape))
```